### PR TITLE
Object detection internal changes

### PR DESCRIPTION
**Pull request by pkulzc**

### CODE REVIEW

File: research/object_detection/meta_architectures/faster_rcnn_meta_arch_test.py [hunk 1]
```diff
@@ -15,14 +15,16 @@
 
 """Tests for object_detection.meta_architectures.faster_rcnn_meta_arch."""
 
+from absl.testing import parameterized
```
Comment (derekjchow) [inline research/object_detection/meta_architectures/faster_rcnn_meta_arch_test.py]: This looks like a new dependency. Should we update the installation docs?

File: research/object_detection/model_main.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```
Comment (derekjchow) [inline research/object_detection/model_main.py]: Copyright 2018

File: research/object_detection/meta_architectures/faster_rcnn_meta_arch_test.py (hunk 1 above)
Comment (pkulzc) [reply research/object_detection/meta_architectures/faster_rcnn_meta_arch_test.py]: Actually I'm able to run this test externally without installing anything else, so I guess it should be fine.

File: research/object_detection/model_main.py (hunk 2 above)
Comment (pkulzc) [reply research/object_detection/model_main.py]: Done, nice catch :)

**Comment by derekjchow** [comment]
LGTM

**Pull request merged**